engine/src/cmd/tests/csv_tests.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gtest/gtest.h>
+
+#include "unit_csv_factory.h"
+#include <iostream>
+#include <fstream>
+#include <iterator>
+
+TEST(CSV, Sanity) {
+    // This may not work for all deployments.
+    // Consider standardizing this.
+    std::ifstream ifs ("../../data/units/units.csv", std::ifstream::in);
```

Comment:
Yeah, I think we might need to figure out something so that this works more reliably across different machines and for different users.

Reply:
I agree but have no idea at the moment. Hopefully, this is an excellent task for a beginner. Well encapsulated and fairly easy.